e2e/heapIgnore.spec.js

```diff
@@ -0,0 +1,87 @@
+require('coffeescript').register();
+_ = require('lodash');
+assert = require('should/as-function');
+
+nodeUtil = require('util');
+testUtil = require('../../heap/test/util');
+rnTestUtil = require('./rnTestUtilities');
+
+const doTestActions = async () => {
+  // Open the HeapIgnore tab in the tab navigator.
+  await element(by.id('HeapIgnore')).tap();
+
+  await expect(element(by.id('totallyIgnored'))).toBeVisible();
+  await element(by.id('totallyIgnored')).tap();
+  await element(by.id('totallyIgnoredHoc')).tap();
+  await element(by.id('allowedInteraction')).tap();
+  await element(by.id('allowedInnerHierarchy')).tap();
+  await element(by.id('allowedAllProps')).tap();
+
+  // :HACK: Break up long URL.
+  // :TODO: Remove once pixel endpoint is handling larger events again.
+  console.log('Waiting 15s to flush iOS events.');
+  await new Promise(resolve => setTimeout(resolve, 15000));
+  await element(by.id('allowedTargetText')).tap();
+  await element(by.id('ignoredTargetText')).tap();
+
+  await element(by.id('heapignoreSentinel')).tap();
+};
+
+describe('HeapIgnore', () => {
+  before(done => {
+    db.orm.connection.sharedRedis().flushall(done);
+  });
+
+  before(async () => {
+    await doTestActions();
+    await rnTestUtil.pollForSentinel('HeapIgnore');
+  });
+
+  it('should ignore the interaction', async () => {
+    const { err, res } = await new Promise((resolve, reject) => {
```

Comment:
Worth a comment saying what you're doing here.  I had to read it a couple of times to make sure I wasn't missing something!

Reply:
done